src/main/java/WayofTime/bloodmagic/command/sub/SubCommandBind.java

```diff
@@ -3,25 +3,35 @@
 import WayofTime.bloodmagic.core.data.Binding;
 import WayofTime.bloodmagic.iface.IBindable;
 import WayofTime.bloodmagic.util.helper.BindableHelper;
-import WayofTime.bloodmagic.util.helper.TextHelper;
-import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommandSender;
-import net.minecraft.command.PlayerNotFoundException;
-import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraft.item.ItemStack;
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.util.text.TextComponentTranslation;
+import net.minecraftforge.server.command.CommandTreeBase;
+import net.minecraftforge.server.command.CommandTreeHelp;
+
+public class SubCommandBind extends CommandTreeBase {
+    public String help = new TextComponentTranslation("commands.bloodmagic.bind.help").getFormattedText();
+    public Object info;
+
+    public SubCommandBind() {
+        addSubcommand(new CommandTreeHelp(this));
+    }
+
+    public Object getInfo() {
```

Comment:
Why `Object`. Why not `String`?

Reply:
getInfo() for commands.bloodmagic.network.get is Integer.